src/components/hand-controls.js

```diff
@@ -1,7 +1,7 @@
 var registerComponent = require('../core/component').registerComponent;
 
-var OCULUS_LEFT_HAND_MODEL_URL = 'https://cdn.aframe.io/controllers/oculus-hands/leftHand.json';
-var OCULUS_RIGHT_HAND_MODEL_URL = 'https://cdn.aframe.io/controllers/oculus-hands/rightHand.json';
+var OCULUS_LEFT_HAND_MODEL_URL = 'https://rawgit.com/arturitu/assets/e698020d6f335fb46a6169f8985b4e12297e157d/controllers/oculus-hands/leftHand.json'; // 'https://cdn.aframe.io/controllers/oculus-hands/leftHand.json';
+var OCULUS_RIGHT_HAND_MODEL_URL = 'https://rawgit.com/arturitu/assets/e698020d6f335fb46a6169f8985b4e12297e157d/controllers/oculus-hands/rightHand.json'; // 'https://cdn.aframe.io/controllers/oculus-hands/rightHand.json';
```

Comment:
They definitely shouldn't (also so you don't have to deal with flushing older versions from all the caches in the Internet).  

If you don't use different URLs, then you have to attempt to update the CDN assets and simultaneously the referencing code (and all uses of it, which seems unreasonable given encouragement to pull versions via npm eather than using direct URL).

You've offered to to update the CDN assets and then update the code, so if that is really what you'd like to do, feel free -- but then we are likely to face spurious issues as we did before reverting the prior hands change.